gltf-json/src/buffer.rs

```diff
@@ -100,15 +103,21 @@ pub struct View {
     pub target: Option<Checked<Target>>,
 
     /// Extension specific data.
-    #[serde(default)]
-    pub extensions: extensions::buffer::View,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub extensions: Option<extensions::buffer::View>,
 
     /// Optional application specific data.
     #[serde(default)]
     #[cfg_attr(feature = "extras", serde(skip_serializing_if = "Option::is_none"))]
     pub extras: Extras,
 }
 
+impl View {
+    pub fn byte_offset_default() -> u32 {
```

Comment:
What is this for?

Reply:
This is called from `src/buffer.rs`:
```rust
    /// Returns the offset into the parent buffer in bytes.
    pub fn offset(&self) -> usize {
        self.json.byte_offset as usize
        self.json.byte_offset.unwrap_or_else(
            json::buffer::View::byte_offset_default) as usize
```
You posted a review comment regarding this call site here:
https://github.com/gltf-rs/gltf/pull/191#discussion-diff-214507143R141

The intention here was to give the default value a descriptive name instead of inserting the literal `0` in various places.